riak/pb/riak_dt_pb2.py

```diff
@@ -995,5 +996,5 @@ class DtUpdateResp(_message.Message):
 
 
 DESCRIPTOR.has_options = True
-DESCRIPTOR._options = _descriptor._ParseOptions(descriptor_pb2.FileOptions(), '\n\027com.basho.riak.protobufB\010RiakDtPB')
+DESCRIPTOR._options = _descriptor._ParseOptions(descriptor_pb2.FileOptions(), b('\n\027com.basho.riak.protobufB\010RiakDtPB'))
```

Comment:
Any reason why you used quotes here?  `b'aaa'` should also work.

Reply:
fixed

Reply:
Let's do this for all other bytestrings.

Reply:
fixed